velodyne_driver/src/driver/driver.cc

```diff
@@ -93,88 +91,80 @@ VelodyneDriver::VelodyneDriver(ros::NodeHandle node,
     }
   else
     {
-      ROS_ERROR_STREAM("unknown Velodyne LIDAR model: " << config_.model);
+      RCLCPP_ERROR(this->get_logger(), "unknown Velodyne LIDAR model: " + config_.model);
       packet_rate = 2600.0;
     }
   std::string deviceName(std::string("Velodyne ") + model_full_name);
 
-  private_nh.param("rpm", config_.rpm, 600.0);
-  ROS_INFO_STREAM(deviceName << " rotating at " << config_.rpm << " RPM");
+  this->get_parameter_or("rpm", config_.rpm, 600.0);
+  RCLCPP_INFO(this->get_logger(), deviceName + " rotating at " + std::to_string(config_.rpm) + " RPM");
```

Comment:
Especially in cases where multiple string concatenations are involved, please use the `sprintf` syntax for the logger macros. In this case, a temporary `std::string` object is created for each string concatenation. With the `sprintf` syntax, only a single temporary object is created for all replacements. e.g.

    RCLCPP_INFO(this->get_logger(), "%s rotating at %d RPM", deviceName, config_.rpm);